package.json

```diff
@@ -126,5 +127,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "packageManager": "yarn@3.6.0"
```

Comment:
I don't see the intention for this, as we already have
```
   "engines": {
     "node": "^18.0.0",
     "yarn": "^3.6.0"
   },
```
Is there any reason to add this?